scripts/cubic/bootstrap-fmriprep.sh

```diff
@@ -263,24 +256,49 @@ echo "outputsource=${output_store}#$(datalad -f '{infos[dataset][id]}' wtf -S da
 echo "cd ${PROJECTROOT}" >> code/merge_outputs.sh
 wget -qO- ${MERGE_POSTSCRIPT} >> code/merge_outputs.sh
 
+dssource="${input_store}#$(datalad -f '{infos[dataset][id]}' wtf -S dataset)"
+pushgitremote=$(git remote get-url --push output)
+
 
 ################################################################################
-# SGE SETUP START - remove or adjust to your needs
+# SLURM SETUP START - remove or adjust to your needs
 ################################################################################
-env_flags="-v DSLOCKFILE=${PWD}/.SGE_datalad_lock"
-echo '#!/bin/bash' > code/qsub_calls.sh
+env_flags="--export=DSLOCKFILE=${PWD}/.SLURM_datalad_lock"
+
+cat > code/sbatch_array.sh <<EOF
+#!/bin/bash
+#SBATCH --job-name=fp_test
+#SBATCH --partition=gablab
+##SBATCH --mem=25GB
+#SBATCH --time=20:00
+##SBATCH --time=4-00:00:00
+#SBATCH --nodes=1
+#SBATCH --cpus-per-task=8
+#SBATCH --mem-per-cpu=8G
+
+#SBATCH --output=logs/array_%A_%a.out
+#SBATCH --error=logs/array_%A_%a.err
+
+#SBATCH --export=DSLOCKFILE=${PROJECTROOT}/analysis/.SLURM_datalad_lock,CONDA_PREFIX=/om2/user/djarecka/miniconda
+
+#SBATCH --array=0-1
+
+subjects=(${SUBJECTS})
+sub=\${subjects[\$SLURM_ARRAY_TASK_ID]}
+
+${PROJECTROOT}/analysis/code/participant_job.sh ${dssource} ${pushgitremote} \$sub \$SLURM_JOB_ID
```

Comment:
is the slurm job id sent here different for different subjects?

Reply:
I thought that it's the same for all subject, but I doubled check and it's not...I have 22501860 and 22501861 (instead of just 22501860)